Jellyfin.Api/Helpers/TranscodingJobHelper.cs

```diff
@@ -444,6 +444,10 @@ private void DeleteHlsPartialStreamFiles(string outputFilePath)
             {
                 var audioCodec = state.ActualOutputAudioCodec;
                 var videoCodec = state.ActualOutputVideoCodec;
+                var hardwareAccelerationTypeString = _serverConfigurationManager.GetEncodingOptions().HardwareAccelerationType;
+                HardwareEncodingType? hardwareAccelerationType = string.IsNullOrEmpty(hardwareAccelerationTypeString)
+                    ? null
+                    : (HardwareEncodingType)Enum.Parse(typeof(HardwareEncodingType), hardwareAccelerationTypeString, true);
```

Comment:
Enum.Parse is terribly inefficient. Given that the enum is converted to a string in the API we're better off using const strings.